src/citrine/resources/branch.py

```diff
@@ -100,6 +101,33 @@ def build(self, data: dict) -> Branch:
         branch.project_id = self.project_id
         return branch
 
+    def get_by_root_id(self, *, branch_root_id: Union[UUID, str]) -> Branch:
+        """
+        Given a branch root ID, retrieve the latest version of the branch.
+
+        Parameters
+        ---------
+        branch_root_id:  Union[UUID, str]
+            Unique identifier of the branch root
+
+        Returns
+        -------
+        Branch
+            The latest version of the branch.
+
+        """
+        params = {"root": str(branch_root_id), "version": "latest"}
+        branch = next(self._list_with_params(per_page=1, **params), None)
+        if branch:
```

Comment:
```suggestion
        if branch is None:
```
Probably equivalent in practice, but explicit None comparison is preferred in style guides.

Reply:
Use of truth values is idiomatic Python, and I don't see PEP 8 taking a stance one way or another about their use.